vignettes/WorkingWithAlignedNucleotides.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+title: "Working with aligned nucleotides (WORK-IN-PROGRESS!)"
+author:
+- name: "Hervé Pagès"
+- name: "Halimat C. Atanda"
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.Date(), '%B %d %Y')`"
+package: GenomicAlignments
+output: BiocStyle::html_document
+vignette: >
+  %\VignetteIndexEntry{Working with aligned nucleotides (WORK-IN-PROGRESS!)}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Introduction
+
+This vignette belongs to the `r Biocpkg("GenomicAlignments")` package. It illustrates how to use the package for working with the nucleotide content of aligned reads.
+
+After the reads generated by a high-throughput sequencing experiment have been aligned to a reference genome, the questions that are being asked about these alignments typically fall in two broad categories: **positional only** and **nucleotide-related**.
+
+**Positional only** questions are about the position of the alignments with respect to the reference genome. Note that the position of an alignment is actually better described in terms of genomic ranges (1 range for an alignment with no gaps, 2 or more ranges for an alignment with gaps). Knowing the ranges of the alignments is sufficient to perform common tasks like *read counting* or for *computing the coverage*. *Read counting* is the process of counting the number of aligned reads per gene or exon and is typically performed in the context of a differential analysis. This task can be accomplished with the `summarizeOverlaps` function provided in the package and is explained in details in the "Counting reads with summarizeOverlaps" vignette (also located in this package). *Computing the coverage* is often the preliminary step to peak detection (ChIP-seq analysis) or to a copy number analysis. It can be accomplished with the `coverage` function. See `? coverage-methods` for more information.
```

Comment:
Let's use

```
`?'coverage-methods'`
```

since you need back ticks or quotes in order to get the help page. There should also be no space.